JSDemos/Demos/Autocomplete/Overview/description.md

```diff
@@ -8,21 +8,27 @@ Use one of the following properties to supply data to the component:
 Accepts a local data array.
 
 * [dataSource](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#dataSource)            
-Accepts a local data array, [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or DevExtreme data store. In this demo, every Autocomplete component is bound to a local array, except the "Custom Item Template and Data Source Usage" component that uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
+Accepts a local data array, [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or DevExtreme data store. In this demo, every Autocomplete component is bound to a local array, except the following components:
+
+    - "Custom Item Template and Data Source Usage"    
+    Uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
+    
+    - "With Custom Store and Search Options"    
```

Comment:
with?

Reply:
Would "Custom Store and Search Options" be better?

Reply:
Absolutely. Please make this edit everywhere else this name appears.